src/views/BrowseView.tsx

```diff
@@ -52,7 +67,9 @@ const Container = styled.div`
   position: relative;
   padding-top: ${sizes(14)};
 `
-
+const Target = styled.div`
```

Comment:
I think it's a good time to move those out to `BrowseView.style.tsx`. Also, maybe let's call this one `IntersectionTarget` for clarity